### ISSUE TITLE
handle error in createAction

### ISSUE THREAD

**Issue by chentsulin** [acdlite/redux-actions#14]
I already checked out createAction file for more information, but it seems don't have any mechanisms to handle error with FSA error action in `createAction` function. It can't mutate any values outside of payload.

But I thought that it's a common case to handle error in a actionCreator, like below:
```
// With babel - optional: ["runtime"]
export const search = createAction(SEARCH_USER, async username => {
  try {
    const data = await getGithubInfo(username);
    return data;
  } catch (err) {
    // handle error here
  }
});
```
It doesn’t make sense to abort error handling support here.

Any ideas?

**Comment by ngokevin** [comment]
Made an attempt https://github.com/acdlite/redux-actions/pull/16

**Comment by lukewestby** [comment]
@chentsulin #16 is merged, can you try running your code agains the master branch? Let us know what you find.

**Comment by timche** [comment]
As this feature has been merged with #16, I will close this issue.
